doc/src/documentation/guides/local-development/README.md

```diff
@@ -1,5 +1,59 @@
 # Local Development
+This page should make it easier to start developing locally.
 
+## Stack folder
+The Repository contains a `stack` folder.
+In this stack folder you can find a docker compose file.
+This docker compose file is configured to deploy all the necessary services
+for the application.
+There is also a folder with keycloak migrations and `.env` files for the different services.
+
+
+### Docker Compose File
+| Service           | Description                                            |
+|-------------------|--------------------------------------------------------|
+| pp-frontend       | The frontend if deployed                               |
+| pp-backend        | The backend if deployed                                |
+| postgres-backend  | Postgres Database for the application                  |
+| keycloak          | Local SSO service                                      |
+| init-keycloak     | Keycloak initializer to apply the keycloak migrations  |
+| postgres-keycloak | Postgres Database for the keycloak service             |
+| postgres-pg-admin | PgAdmin to access the database                         |
+| mail-maildev      | Local Mail-Client                                      |
+| mail-mailpit      | Local SMTP-Server                                      |
+
+Use docker compose to start the infrastructure
+
+```
+# Starts the infrastructure services like keycloak and db for backend
+docker compose up -d
+
+# Starts all services
+docker compose --profile full up -d
+
+# Includes the backend service on startup
+docker compose --profile backend up -d
+
+# Includes the frontend service on startup
+docker compose --profile frontend up -d
+```
+
+The frontend and Backend are only started if docker compose is run with the profile full.
+If you want to develop locally it is sugessted to start docker compose without this profile
+and start those parts via your IDE.
+The `.env` files for the frontend and backend are used when these services are deployed.
+Ich you start them via your IDE they are not used.
+
+### Keycloak
+An admin user and a test user are created by `init-keycloak`. You can change the configuration via the keycloak ui.
```

Comment:
Maybe include the Users that are added at this point.

Reply:
discarded. I think it is very risky to have this part up date. When we change or add other users in migration we have to change the documentation too. The current description already tells where and how to find the users: look into `keycloak/migration` directory. (no the most friendly solution but an acceptable solution)